tools/Google.Cloud.Tools.ReleaseManager/History/GitCommit.cs

```diff
@@ -62,6 +75,21 @@ internal IEnumerable<string> GetHistoryLines()
                 messageLines = messageLines.TakeWhile(line => !line.StartsWith("PiperOrigin-RevId")).ToList();
             }
 
+            // Allow the version history to be overridden by a line on its own of "Version history:"
+            // followed by all the lines that should be in version history.
+            int overrideIndex = messageLines.IndexOf(VersionHistoryOverride);
+            if (overrideIndex != -1)
+            {
+                messageLines = messageLines.Skip(overrideIndex + 1).ToList();
+            }
+
+            // A single line of "skip" (usually from an override, either in the commit or in the overrides file)
+            // means we skip the commit entirely.
+            if (messageLines.Count == 1 && messageLines[0].Equals("skip", StringComparison.OrdinalIgnoreCase))
```

Comment:
Maybe add a constant for skip VersionHistorySkip, but that can be done later.

Reply:
Added it as SkipMessage now. Will merge on green.